### PR TITLE
initial commit of Exium xe-cnf package

### PR DESCRIPTION
**Pull request by umanathlanka**
Initial commit of Exium xe-cnf package

### CODE REVIEW

File: applications/xe-cnf/.gitignore [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# These are some examples of commonly ignored file patterns.
```
Comment (abjyoti) [inline applications/xe-cnf/.gitignore:1]: This file doesn't have Copyright header. This is git configuration file. There is no need to have this file fin repo. Can you please remove this file?

File: applications/xe-cnf/helm/exe-health/.helmignore [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```
Comment (abjyoti) [inline applications/xe-cnf/helm/exe-health/.helmignore:1]: This file doesn't have Copyright header. This is helm configuration file. What is the need to have this file in repo. Can you please remove this file?

File: applications/xe-cnf/helm/n3iwf-eap5g/.helmignore [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```
Comment (abjyoti) [inline applications/xe-cnf/helm/n3iwf-eap5g/.helmignore:1]: This file doesn't have Copyright header. This is helm configuration file. There is no need to have this file in repo. Can you please remove this file?

File: applications/xe-cnf/helm/n3iwf-n2fe/.helmignore [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```
Comment (abjyoti) [inline applications/xe-cnf/helm/n3iwf-n2fe/.helmignore:1]: This file doesn't have Copyright header. This is helm configuration file. There is no need to have this file in repo. Can you please remove this file?

File: applications/xe-cnf/lib/redis-pv-master.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```
Comment (abjyoti) [inline applications/xe-cnf/lib/redis-pv-master.yaml:1]: Missing Copyright headers.

File: applications/xe-cnf/lib/redis-pv-slave.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```
Comment (abjyoti) [inline applications/xe-cnf/lib/redis-pv-slave.yaml:1]: Missing Copyright headers.

**Review by abjyoti — COMMENTED** [review]
Please resolve the review comments

**Comment by umanathlanka** [comment]
Comments are reviewed and addressed missing license headers.

**Comment by archie951** [comment]
> Comments are reviewed and addressed missing license headers.
Hi, The Travis build has failed and reported few errors which you will need to fix. 
https://app.travis-ci.com/github/open-ness/edgeapps/builds/232579835
Request you to please address these.

File: applications/xe-cnf/config.sh [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 Exium Inc.
+
+#!/bin/bash
```
Comment (archie951) [inline applications/xe-cnf/config.sh:4]: The shebang must be on the first line. 

#!/bin/bash
# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2021 Exium Inc.

Alter it like this

File: applications/xe-cnf/uninstall.sh [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (archie951) [inline applications/xe-cnf/uninstall.sh:1]: Add the shebang on top of the script.
#!/bin/bash

**Review by archie951 — DISMISSED** [review]

File: applications/xe-cnf/.gitignore (hunk 1 above)
Comment (archie951) [reply applications/xe-cnf/.gitignore]: Please remove this file

**Comment by umanathlanka** [comment]
.gitignore file is removed from xe-cnf folder now.

**Review by abjyoti — APPROVED** [review]
I am fine...

**Pull request merged**